portafly/src/i18n/locales/en/accounts.yml

```
@@ -1,20 +1,95 @@
+  name_header: Name
+  name_header_description: Column listing names of team members
+  email_header: Email
+  email_header_description: Column listing the email addresses of team members
+  sso_authentication_heaer: SSO authentication
+  sso_authentication_header_description: Column listing if SSO authentication has been configured for team members
+  [PROPOSED] sso_authentication_value: [Yes, Not yet]
+  role_header: Role
+  role_header_description: Column listing the roles associated to each team member
+  [PROPOSED] role_header_value: [Admin (you), Admin, Member]
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
+## Tab: Invitations
+## Toolbar
+bulk_selector: {{number}} selected
+bulk_selector_description: Label for the bulk selector to indicate the number of chosen invitations. This label is visible only when invitations were selected.
+#-------# Delete button: available in shared.yml
+[PROPOSED] invitations_filter: [Recipient, Acceptance]
+invitations_filter_description: List containing the criteria to filter invitations
+invitations_typeahead: [Filter by recipient, Filter by acceptance]
+invitations_typeahead_description: Filter by typing ahead the text.
+
+invitations_table:
+  aria_label: Invitations table
+  aria_label_description: Aria label of the table of invitations
+  [PROPOSED] empty_state: Currently, there are no invitations sent. Start sending invitations to your team members.
+  empty_state_description: Message to show when there are no invitations to show, and prompt the user to send some invites
+  recipient_header: Recipient
+  recipient_header_description: Column listing the email addresses of invitation recipients
+  [PROPOSED] sent_header: Sent on UTC time
+  sent_header_description: Column listing the date when invitations were sent
+  [PROPOSED] acceptance_header: Accepted on UTC time
+  acceptance_header_description: Column listing the date when the invitations were accepted
+  [PROPOSED] acceptance_values: [Yes - UTC time, Not yet]
+  actions_header: Actions
+  actions_header_description: Column listing the available actions to perform with each invitation
+
+# Modal: Invitation deletion
+invitation_deletion:
+  [PROPOSED] invitation_deletion_page: Delete selected invitations
+  invitation_deletion_page_description: Body title of modal dialog to delete the selected invitations from the table
+  [PROPOSED] admonition: This action will irreversibly delete the selected invitations. Please confirm below to proceed with the deletion.
+  admonition_description: Admonition text to explain that the removal of the selected invitations cannot be undone.
+  #-------# Delete button: available in shared.yml
+  #-------# Cancel button: available in shared.yml
+
+# Modal: Invitation to a new member
+invitation_sending:
+  invitation_sending_page: Invite a new team member
```

I think including the "section" prefix is a bit redundant. Either use prefixes:
```suggestion
invitation_sending_page: Invite a new team member
```
Or organize by sections:
```suggestion
invitation_sending:
  page: Invite a new team member
```

---

I'll test the alternatives and submit the changes.

---

@josemigallas changes are ready for your review: https://github.com/3scale/porta/pull/1861/commits/a723442a74fce9ed9f67177c84a3079603041854